src/diffusers/models/autoencoder_kl.py

```diff
@@ -121,6 +153,9 @@ def encode(self, x: torch.FloatTensor, return_dict: bool = True) -> AutoencoderK
         return AutoencoderKLOutput(latent_dist=posterior)
 
     def _decode(self, z: torch.FloatTensor, return_dict: bool = True) -> Union[DecoderOutput, torch.FloatTensor]:
+        if self.use_tiling and (z.shape[-1] > 64 or z.shape[-2] > 64):
```

Comment:
```suggestion
        if self.use_tiling and (z.shape[-1] > 64 or z.shape[-2] > 64):
            # enable tiling only if latent image size is larger than 64 as otherwise no memory is saved
```